openprescribing/openprescribing/settings/production.py

```diff
@@ -83,15 +83,23 @@
             'formatter': 'verbose',
             'filename': "%s/logs/mail-signals.log" % REPO_ROOT,
             'maxBytes': 1024 * 1024 * 100,  # 100 mb
-        }
-
+        },
+        'sentry': {
+            'level': 'WARNING',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler'
+        },
     },
     'loggers': {
         'django': {
             'handlers': ['gunicorn'],
             'level': 'WARN',
             'propagate': True,
         },
+        'django.security.csrf': {
```

Comment:
Didn't know about this.

Reply:
Neither did I until today. But it struck me as really terrible that there's a special magic string you can forget to put in a form and then it just won't work but will never trigger an error so I thought it was worth seeing if Sentry could warn us about it. And it turns out it can quite easily.